packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1530,29 +1528,11 @@ abstract class RenderObjectElement<T extends RenderObjectWidget> extends Buildab
   }
 
   void performRebuild() {
-    reinvokeBuilders();
+    assert(false); // Currently unreachable.
+    widget.updateRenderObject(renderObject, widget);
     _dirty = false;
   }
 
-  void reinvokeBuilders() {
-    // There's no way to mark a normal RenderObjectElement dirty.
-    // We inherit from BuildableElement so that subclasses can themselves
-    // implement reinvokeBuilders() if they do provide a way to mark themeselves
-    // dirty, e.g. if they have a builder callback. (Builder callbacks have a
-    // 'BuildContext' argument which you can pass to Theme.of() and other
-    // InheritedWidget APIs which eventually trigger a rebuild.)
```

Comment:
I remember now. This comment is how you end up here. Suppose your MixedViewport has a builder function that does `Foo.of(context)`. The `context` here will be this Element.